Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.h

```diff
@@ -100,6 +100,11 @@ class WebFullScreenManager final : public WebCore::EventListener {
     Ref<WebPage> m_page;
     RefPtr<WebCore::Element> m_element;
     WeakPtr<WebCore::Element, WebCore::WeakPtrImplWithEventTargetData> m_elementToRestore;
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+    WebCore::FloatSize m_oldSize;
+    double m_scaleFactor;
+    double m_minEffectiveWidth;
```

Comment:
Let's give these initial values, to avoid potential UB in the future.

```suggestion
    double m_scaleFactor { 1 };
    double m_minEffectiveWidth { 0 };
```